spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -1708,14 +1708,12 @@ public void testDefaultCopyOnWriteMergePartitionedSortedTable() {
 
     table.replaceSortOrder().desc("id").commit();
 
-    Expression[] expectedClustering = new Expression[] {Expressions.identity("date")};
-    Distribution expectedDistribution = Distributions.clustered(expectedClustering);
-
     SortOrder[] expectedOrdering =
         new SortOrder[] {
           Expressions.sort(Expressions.column("date"), SortDirection.ASCENDING),
           Expressions.sort(Expressions.column("id"), SortDirection.DESCENDING)
         };
+    Distribution expectedDistribution = Distributions.ordered(expectedOrdering);
```

Comment:
Why do we change expected Distribution here? Is this just identical to the above code?

Reply:
The asserted expected distribution is different. 

Before it was clustered on the `date` field, after it is ordered by `date` and `id` fields. 
I've updated this test as otherwise the test fails with following error:
```
    java.lang.AssertionError: Distribution must match 
expected:<ClusteredDistribution(date)> 
but was:<OrderedDistribution(date ASC NULLS FIRST, id DESC NULLS LAST)>
```

Reply:
This would cause a performance regression.